### PR TITLE
⚠️ CONFLICT! Lineage pull request for: skeleton

### PR DESCRIPTION
**Pull request by cisagovbot**
# Lineage Pull Request: CONFLICT #

[Lineage] has created this pull request to incorporate new changes found in an
upstream repository:

Upstream repository: [`https://github.com/cisagov/skeleton-ansible-role.git`](https://github.com/cisagov/skeleton-ansible-role.git)
Remote branch: `HEAD`

Check the changes in this pull request to ensure they won't cause issues with
your project.

The `lineage/skeleton` branch has **one or more unresolved merge conflicts**
that you must resolve before merging this pull request!

## How to resolve the conflicts ##

1. Take ownership of this pull request by removing any other assignees.

1. Clone the repository locally, and reapply the merge:

    ```console
    git clone git@github.com:cisagov/ansible-role-terraform.git ansible-role-terraform
    cd ansible-role-terraform
    git remote add skeleton https://github.com/cisagov/skeleton-ansible-role.git
    git remote set-url --push skeleton no_push
    git switch develop
    git switch --create lineage/skeleton --track origin/develop
    git pull skeleton HEAD
    git status
    ```

1. Review the changes displayed by the `status` command.  Fix any conflicts and
   possibly incorrect auto-merges.

1. After resolving each of the conflicts, `add` your changes to the
   branch, `commit`, and `push` your changes:

    ```console
    git add README.md 
    git commit
    git push --force --set-upstream origin lineage/skeleton
    ```

    Note that you may *append* to the default merge commit message
    that git creates for you, but *please do not delete the existing
    content*.  It provides useful information about the merge that is
    being performed.

1. Wait for all the automated tests to pass.

1. Confirm each item in the "Pre-approval checklist" below.

1. Remove any of the checklist items that do not apply.

1. Ensure every remaining checkbox has been checked.

1. Mark this draft pull request "Ready for review".

## ✅ Pre-approval checklist ##

Remove any of the following that do not apply. If you're unsure about
any of these, don't hesitate to ask. We're here to help!

- [x] ✌️ The conflicts in this pull request have been resolved.
- [x] All relevant type-of-change labels have been added.
- [x] All relevant repo and/or project documentation has been updated
      to reflect the changes in this PR.
- [x] All new and existing tests pass.

---

> **Note**
> You are seeing this because one of this repository's maintainers has
> configured [Lineage] to open pull requests.

For more information:

🛠 [Lineage] configurations for this project are stored in `.github/lineage.yml`

📚 [Read more about Lineage][Lineage]

[//]: # (lineage:metadata:{&quot;slayed&quot;:true})
[Lineage]: https://github.com/cisagov/action-lineage/ "Lineage GitHub Action"

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -15,10 +15,10 @@ None.
...
 | Variable | Description | Default | Required |
 |----------|-------------|---------|----------|
-| group | The group associated with the Terraform and terraform-docs binaries. | `root` | No |
-| mode | The mode of the Terraform and terraform-docs binaries. | `0755` | No |
-| owner | The owner of the Terraform and terraform-docs binaries. | `root` | No |
-| install\_dir | The directory where the Terraform and terraform-docs binaries are to be installed. | `/usr/local/bin` | No |
+| terraform\_group | The group associated with the Terraform and terraform-docs binaries. | `root` | No |
+| terraform\_mode | The mode of the Terraform and terraform-docs binaries. | `0755` | No |
+| terraform\_owner | The owner of the Terraform and terraform-docs binaries. | `root` | No |
+| terraform\_install\_dir | The directory where the Terraform and terraform-docs binaries are to be installed. | `/usr/local/bin` | No |
```
Comment (dav3r) [inline README.md]: Put variables in alphabetical order:
```suggestion
| terraform\_group | The group associated with the Terraform and terraform-docs binaries. | `root` | No |
| terraform\_install\_dir | The directory where the Terraform and terraform-docs binaries are to be installed. | `/usr/local/bin` | No |
| terraform\_mode | The mode of the Terraform and terraform-docs binaries. | `0755` | No |
| terraform\_owner | The owner of the Terraform and terraform-docs binaries. | `root` | No |
```

File: defaults/main.yml [hunk 2]
```diff
@@ -1,13 +1,13 @@
 ---
 # The directory where the Terraform and terraform-docs binaries are to
 # be installed
-install_dir: /usr/local/bin
+terraform_install_dir: /usr/local/bin
 # The group of the Terraform and terraform-docs binaries
-group: root
+terraform_group: root
 # The mode of the Terraform and terraform-docs binaries
```
Comment (dav3r) [inline defaults/main.yml]: The order of these two variables should be reversed to keep them sorted alphabetically.

**Review by dav3r — CHANGES_REQUESTED** [review]

File: tasks/main.yml [hunk 3]
```diff
@@ -9,24 +9,24 @@
...
     # The archive also includes a README and a LICENSE file, but we
     # don't want those.
     include: terraform-docs
-    mode: "{{ mode }}"
-    owner: "{{ owner }}"
+    mode: "{{ terraform_mode }}"
+    owner: "{{ terraform_owner }}"
     remote_src: yes
     # ansible-lint generates a warning that "lines should be no longer
     # than 160 characters" here", but there is no good way to break up
```
Comment (jsf9k) [inline tasks/main.yml]: Can we change the `noqa 204` below to use the rule name instead?

**Review by jsf9k — CHANGES_REQUESTED** [review]

File: tasks/main.yml (hunk 3 above)
Comment (jmorrowomni) [reply tasks/main.yml]: @jsf9k I removed the `noqa` code in order to get ansible-lint to hopefully spit out the rule name, and ansible-lint didn't complain at all....should I just remove the ignore directive?

Comment (jsf9k) [reply tasks/main.yml]: Yes, please do that.

File: README.md (hunk 1 above)
Comment (jmorrowomni) [reply README.md]: Addressed in [09c15c2](https://github.com/cisagov/ansible-role-terraform/pull/24/commits/09c15c20497c773232ce68946159ed43ba60b386) - also moved terraform_docs_version

File: defaults/main.yml (hunk 2 above)
Comment (jmorrowomni) [reply defaults/main.yml]: Addressed in [09c15c2](https://github.com/cisagov/ansible-role-terraform/pull/24/commits/09c15c20497c773232ce68946159ed43ba60b386) - also moved terraform_docs_version

File: tasks/main.yml (hunk 3 above)
Comment (jmorrowomni) [reply tasks/main.yml]: Done in [69271e1](https://github.com/cisagov/ansible-role-terraform/pull/24/commits/69271e160e2a1d050e1ba246615cdf27f557f16f)

**Review by dav3r — APPROVED** [review]

**Review by jsf9k — APPROVED** [review]

**Pull request merged**